AlloyDb/VERSION

```diff
@@ -0,0 +1 @@
+0.0.0
```

Comment:
is this expected to not be `0.0.1`?

Reply:
not yet! The release script will update it to be `0.0.1` on release. Before it has a release, it is correct to be `0.0.0`